task.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+trait ChangeAttributes {
+    public function capitalize() {
+        foreach ($this->attributes as $attrName => $attrValue) {
+            $this->attributes[$attrName] = strtoupper($attrValue);
+        }
+    }
+}
+
+abstract class Entity
+{
+    //Приватное свойство
+    private $attributes;
+    
+    //Можно задать атрибуты через конструктор
+    public function __construct($attributes = array())
+    {
+        $this->attributes = $attributes;
+    }
+    
+    //Задать атрибут обьекта
+    public final function setAttr($attrName, $value)
+    {
+        $this->attributes[$attrName] = $value;
+        return true;
+    }
+    
+    //Получить атрибут обьекта
+    public final function getAttr($attrName, $defaultValue=null)
+    {
+        if(isset($this->attributes[$attrName])) {
+            return $this->attributes[$attrName];
+        }
+        
+        return $defaultValue;
+    }
+    
+    //Получить атрибут обьекта
+    public final function delAttr($attrName)
+    {
+        if(isset($this->attributes[$attrName])) {
+          unset($this->attributes[$attrName]);
+        }
+        return true;
+    }
+    
+    //Вывести информацию по всем заданым атрибутам
+    public function printInfo()
+    {
+        echo "<table>";

```

Comment:
don't echo any information in your classes. Let this method return the string, and output the result in your index.php file